### DISCUSSION TITLE
.co.uk DNS_PROBE_FINISHED_NXDOMAIN

### DISCUSSION THREAD

**Discussion by bubblemanmaxwell** [mozilla/hubs#3979] [Hubs Cloud Questions] [answered]
Hello. I am having problems with recipe 3 , with .co.uk site. 
I have tried to follow all docs but keep finding problems. their is little about .co.uk known bug.

what I have done so far.
1)bought 2 domains on route 53. 1) .co.uk  2) .link
2) had ssl certificates validated in my region (Eu east) and us-east 1. for my (.co.uk site.) 
i have ssl cert for eu east for my (.link) 
3) I have launched my stack successfully

when I go to my url....  i have  DNS_PROBE_FINISHED_NXDOMAIN

Im getting assistance form aws but maybe this is more Mozilla hubs.

The videos on how to deploy are so helpful. but it would be great if they covered recipe 3

Thank you for any help.
@robinkwilson

**Comment by johnshaughnessy** [comment, marked as answer]
Do any of the previous discussions about `DNS_PROBE_FINISHED_NXDOMAIN` contain the solution to your problem?

https://github.com/mozilla/hubs/discussions/3489
https://github.com/mozilla/hubs/discussions/3277
https://github.com/mozilla/hubs/discussions/3028
https://github.com/mozilla/hubs/discussions/2745

**Comment by pattersonbl2** [comment]
are you still having issues with this ?
